x-pack/test/oidc_api_integration/apis/implicit_flow/oidc_auth.ts

```diff
@@ -31,22 +33,30 @@ export default function({ getService }: FtrProviderContext) {
       it('should return an HTML page that will parse URL fragment', async () => {
         const response = await supertest.get('/api/security/v1/oidc/implicit').expect(200);
         const dom = new JSDOM(response.text, {
+          url: formatURL({ ...config.get('servers.kibana'), auth: false }),
```

Comment:
__note:__ we need a bit more magic for external script :see_no_evil: But I still like it more than functional tests (even though they would have caught issues with changed CSP I guess, assuming we could reproduce full OIDC flow in the functional test).